algorithms/appo/learner.py

```diff
@@ -109,6 +109,19 @@ def __init__(
 
         self.process = Process(target=self._run, daemon=True)
 
+        if is_continuous_action_space(self.action_space) and self.cfg.exploration_loss == 'symmetric_kl':
+            raise NotImplementedError(f'KL-divergence exploration loss is not supported with '
```

Comment:
String literals start with an f'' if you're not formatting anything :) Just a minor thing

Reply:
Done